README.md

```diff
@@ -37,8 +37,48 @@ sudo npm run patch:hosts
 
 Update `fec.config.json` according to your application URL. [Read more](http://front-end-docs-insights.apps.ocp4.prod.psi.redhat.com/ui-onboarding/fec-binary#Configuration)[Older webpack config](https://github.com/RedHatInsights/frontend-components/tree/master/packages/config#useproxy).
 
+### Setting up dependencies
+
+The apps config is downloaded from environment you run in,
+but you can stub that by running cloud-services-config locally
+
+The guide for running it is at https://github.com/RedHatInsights/cloud-services-config#testing-your-changes-locally.
+
+To run with another app you can easily run one app just for serving static assets and the other as the main app.
+Pointing the asset route to the statically served assets of the second app.
+
+#### Image builder with Provisioning static assets
+
+
+Run Provisioning frontend with `npm run start:federated`.
```

Comment:
it fails on my env without specifying `FEC_CONFIG_PATH`, should we add `--config` flag with the config path?

Reply:
it should have been fixed by https://github.com/RedHatInsights/frontend-components/pull/1631 I guess we should bump our dependency and then it would be fine? but I'm fine adding the config flag :)